core/src/main/java/hudson/model/ManagementLink.java

```diff
@@ -122,6 +122,16 @@ public boolean getRequiresConfirmation() {
         return Jenkins.ADMINISTER;
     }
 
+    /**
+     * Whether to show this link on /manage
+     *
+     * @return true if and only if the link should be shown
+     */
+    public boolean isShowLink() {
+        final Permission requiredPermission = getRequiredPermission();
+        return requiredPermission == null || Jenkins.get().hasPermission(requiredPermission);
```

Comment:
I beleive this is incorrect with the rephrasing of the javadoc for `getRequiredPermission`

Reply:
there is a lot of boilerplate repeated here, wonder if the base class could be changed to use have `public Permission[] getRequiredPermissions() {...}` which defaulted to calling `getRequiredPermission()` then all we need to call is `hasAnyPermission(getRequiredPermissions())`?